Notation.Plugin.AzureKeyVault/Protocol/KeySpec.cs

```diff
@@ -66,6 +79,14 @@ public static KeySpec ExtractKeySpec(X509Certificate2 signingCert)
 
         /// <summary>
         /// Encodes the key spec to be string.
+        /// Supported key types are RSA with key size 2048, 3072, 4096
+        /// and ECDSA with key size 256, 384, 521.
+        ///
+        /// <param name="keySpec">The key spec to be encoded</param>
+        ///
+        /// <returns>
+        /// The encoded key spec, including RSA-2048, RSA-3072, RSA-4096, EC-256, EC-384, EC-521
```

Comment:
indent or not indent? consistency is a question.

Reply:
Updated to remove the indent.